### DISCUSSION TITLE
Is it possible to specify a list of filers for the weed s3 -filler option?

### DISCUSSION THREAD

**Discussion by aoberest** [seaweedfs/seaweedfs#2574] [General]
Hello!
I splited the functionality the weed filter and weed s3 into different nodes.
I have two `weed filer` nodes and two `weed s3` nodes. 

Start `/usr/bin/weed s3 -options=/etc/seaweedfs/s3.conf`

s3.conf

```
allowEmptyFolder=true
cert.file=/etc/seaweedfs/ssl/s3.crt
key.file=/etc/seaweedfs/ssl/s3.key
domainName=s3.test.com
filer=10.193.85.151:8888,10.193.85.152:8888
ip.bind=10.193.85.148
config=/var/lib/seaweedfs/filer/config.json
port=443
```

And got message:

`
weed-s3[42055]: I0111 15:35:01 42055 s3.go:167] wait to connect to filer 10.193.85.151:8888,10.193.85.152:8888 grpc address [10.193.85.151:8888,10.193.85.152]:18888
`

Is it possible to specify a list of filers for the weed s3 -filler option?
Can this feature be added in future releases?

**Comment by chrislusf** [comment]
This is not recommended. Better put "weed s3" and "weed filer" in pairs.
